### PR TITLE
RTL support for OMV UI

### PR DESCRIPTION
**Pull request by myomv100**
Hello 

This quick UI addition for the languages that are RTL.

### CODE REVIEW

**Comment by myomv100** [comment]
I think the implementation could be better. This might be taken as a reference to what need to be done to support RTL. I might  redo it in a cleaner way soon.

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -47,7 +47,7 @@ Vagrant.configure("2") do |config|
 	config.vm.provider :libvirt do |prov|
 		prov.memory = "2048"
 		# Add some disks.
-		for i in 0..2 do
+		for i in 0..1 do
```
Comment (votdev) [inline vagrant/Vagrantfile]: See above

File: deb/openmediavault/var/www/openmediavault/css/theme-triton-rtl.scss [hunk 2]
```diff
@@ -0,0 +1,292 @@
+/**
```
Comment (votdev) [inline deb/openmediavault/var/www/openmediavault/css/theme-triton-rtl.scss]: Please extract only the necessary changes here. This way there is no duplicate code and the changes that are needed for RTL are visible.

File: deb/openmediavault/var/www/openmediavault/css/theme-all-rtl.scss [hunk 3]
```diff
@@ -0,0 +1,624 @@
+/**
```
Comment (votdev) [inline deb/openmediavault/var/www/openmediavault/css/theme-all-rtl.scss]: Please extract CSS that is necessary for RTL. Prevent duplicate code.

**Pull request closed without merging**